src/main/cpp/Robot.cpp

```diff
@@ -82,6 +95,65 @@ void Robot::ExpectAutonomousEndConds() {
     }
 }
 
+void Robot::ClimbingSequenceSM() {
+    switch (m_state) {
+        case ClimbingStates::kSecondRung: {
+            solenoidTimer.Start();
+            if (solenoidTimer.HasElapsed(1_s)) {
+                climber.TelescopingOut();
+                solenoidTimer.Stop();
+                solenoidTimer.Reset();
+            } else if ((!climber.BarSensor()) && (!climber.UppperSensor())) {
+                climber.TelescopingExtention(0.65);
+            }
```

Comment:
There should be an else if statement so that the robot stops moving when the the upper sensor is active